macros/src/contract.rs

```diff
@@ -180,4 +186,36 @@ pub mod utils {
             }
         }
     }
+
+    pub fn find_method<'a>(
+        input: &'a ContractTrait,
+        method_name: &'static str,
```

Comment:
I checked and it is possible to remove `static`, so it looks like: `method_name: &str,`.